command/agent/config.go

```diff
@@ -132,6 +132,15 @@ type Config struct {
 	// allows Serf agents to join each other with zero configuration.
 	Discover string `mapstructure:"discover"`
 
+	// SRVRecords is used look for other agents using DNS SRV records.
+	// When this is set, the agent will look up the SRV record
+	// and attempt to add any hosts it finds. You can specify multiple times
+	// to look up multiple SRV records.
+	JoinSRV []string `mapstructure:"join-srv"`
+
+	// Exactly like join-srv, only keep trying until a successful join happens.
+	RetryJoinSRV []string `mapstructure:"retry-join-srv"`
+
```

Comment:
These should probably be underscores to follow the config file convention of all of the other options.